crates/driver/src/boundary/score.rs

```diff
@@ -0,0 +1,59 @@
+use {
+    crate::{
+        domain::{
+            self,
+            competition::{self, score},
+            eth,
+        },
+        util::conv::u256::U256Ext,
+    },
+    solver::settlement_rater::{ScoreCalculator, ScoringError},
+};
+
+type SuccessProbability = f64;
```

Comment:
I guess we could use the new type pattern here instead of a type alias since this is already used a lot in the driver.

Reply:
Created new type.